src/main/java/net/earthmc/emcapi/manager/SSEManager.java

```diff
@@ -0,0 +1,42 @@
+package net.earthmc.emcapi.manager;
+
+import com.google.gson.JsonObject;
+import io.javalin.Javalin;
+import io.javalin.http.sse.SseClient;
+
+import java.util.Queue;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+
+public class SSEManager {
+
+    private final Javalin javalin;
+    private final static Queue<SseClient> clients = new ConcurrentLinkedQueue<>();
+
+    public SSEManager(Javalin javalin) {
+        this.javalin = javalin;
+    }
+
+    public void loadSSE() {
+        javalin.sse("v3/events", client -> {
+            client.keepAlive();
+            client.sendEvent("open", "Connected to the EarthMC API.");
+            client.onClose(() -> clients.remove(client));
+            clients.add(client);
+        });
+    }
+
+    public static void broadcastMessage(String event, JsonObject data) {
+        int timestamp = Math.toIntExact(System.currentTimeMillis() / 1000);
+        data.addProperty("timestamp", timestamp);
+        String message = data.toString();
+
+        CompletableFuture.allOf(
+                clients.stream()
+                        .map(client -> CompletableFuture.runAsync(() -> client.sendEvent(event, message)))
+                        .toArray(CompletableFuture[]::new)
+        );
```

Comment:
This can be simplified, the result of the futures is ignored so combining them into one is redundant.

(requires adding a plugin field obviously)
```suggestion
        for (final SseClient client : clients) {
        	plugin.getServer().getAsyncScheduler().runNow(plugin, task -> client.sendEvent(event, message));
        }
```